lms/static/sass/discussion/views/_create-edit-post.scss

```diff
@@ -87,9 +88,9 @@
 
   .post-type-label {
     @extend %cont-truncated;
-    @include box-sizing(border-box);
+    box-sizing: border-box;
     @include white-button;
-    @include font-size(14);
+    font-size: 14px;
```

Comment:
@dianakhuang When I originally had these styles working in the Pattern Library the includes were not available, so I thought it was simpler to just rewrite them. Since then I've added the include to the file _compatibility-v1.scss, so I think I'll revert these changes.

@clrux I agree that the font usages need cleaning up throughout forums. This PR is just intended to get the forums styled as-is working in the Pattern Library, so I've reported a new story to do this clean up:

https://openedx.atlassian.net/browse/TNL-4708